js/components/content.tsx

```diff
@@ -169,13 +183,13 @@ class Content extends React.Component<IComponentProps, IContentState> {
         }
         // Ensure the new styles are applied and we update state for any text modifications made by the user
         // between presenting the <textarea> UI and initializing tinyMCE
+
         this.editor.fire('change');
     }
 
     onChange = (e) => {
         const contentValue = this.editor.getContent();
         manywho.state.setComponent(this.props.id, { contentValue }, this.props.flowKey, true);
-        this.forceUpdate();
```

Comment:
Are we sure this is not needed? Most other components that call setComponent then either force a re-render or trigger a call to the engine which in turn triggers a re-render. I only mention it as every time I have removed a forceUpdate it has caused some kind of regression.

Reply:
Not as far as I can tell - leaving it in made TinyMCE act a bit crazy with the new collaboration support. I can't remember whether it was causing the text to render in reverse (as the collaboration event would make the selection go back to the start) or whether it was stripping whitespace from the end, but it was one of the two.

The component state is still updated as usual in the UI, but it's just not re-rendered by React. TinyMCE is still handling the DOM updates itself though.